modules/caddyhttp/tracing/tracer.go

```diff
@@ -108,8 +108,8 @@ func (ot *openTelemetryWrapper) newResource(
 	webEngineVersion string,
 ) (*resource.Resource, error) {
 	return resource.Merge(resource.Default(), resource.NewSchemaless(
-		semconv.WebEngineNameKey.String(webEngineName),
-		semconv.WebEngineVersionKey.String(webEngineVersion),
+		semconv.WebEngineName(webEngineName),
+		semconv.WebEngineVersion(webEngineVersion),
```

Comment:
The old way is still functional, but I spotted a slight API improvement in `v1.13.0` so I made the change.